dev-requirements.txt

```diff
@@ -24,14 +24,13 @@ aiohttp
 
 #### DOC-RELATED PACKAGES ####
 
-# Builds our documentation.
-sphinx>=2.1.1
+# YAML manipulation
+ruamel.yaml
 
-# Watches the documentation directory and rebuilds on changes.
-sphinx-autobuild
-
-# doc theme
-sphinx_rtd_theme
+pydoc-markdown==2.0.5
```

Comment:
Do these need to be pinned?

Reply:
These aren't user requirements, so it's less important that they are unpinned. We should try using a bot to upgrade some of these dependencies.